src/components/NewBook.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBookAsync } from '../redux/books/books';
+
+const NewBook = () => {
+  const dispatch = useDispatch();
+  return (
+    <form>
+      <h2>Add new Book</h2>
+      <div className="inputContainer">
+        <input id="bookTitleInput" required placeholder="Book Title" />
+        <input id="bookAuthorInput" required placeholder="Author" />
+        <button
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a feature that cleans your **inputs form** once a user added a new book. Right now, the old text remains there while the book has been added.